howso/analysis_weights.amlg

```diff
@@ -422,13 +433,12 @@
 							feature_residuals_map
 								(call !CalculateFeatureResiduals (assoc
 									features features
-									robust_residuals (true)
+									robust_residuals computing_mda
+									superfull (true)
```

Comment:
having 2 binary flags `robust_residuals` and `superfull` will probably be pretty confusing. Maybe a string parameter would be better long term?